bundle/edu.gemini.itc.web/src/main/java/edu/gemini/itc/web/html/NiriPrinter.java

```diff
@@ -52,44 +52,35 @@ private void writeSpectroscopyOutput(final UUID id, final SpectroscopyResult res
 
         _println("");
 
-        // This object is used to format numerical strings.
-        final FormatStringWriter device = new FormatStringWriter();
-        device.setPrecision(2); // Two decimal places
-        device.clear();
-
         // Altair specific section
         if (result.aoSystem().isDefined()) {
             _println(HtmlPrinter.printSummary((Altair) result.aoSystem().get()));
         }
 
         if (!result.observation().isAutoAperture()) {
-            _println("software aperture extent along slit = "
-                    + device.toString(result.observation().getApertureDiameter()) + " arcsec");
+            _println(String.format("software aperture extent along slit = %.2f arcsec", result.observation().getApertureDiameter()));
         } else {
             switch (result.source().getProfileType()) {
```

Comment:
In case you're worried about performance then I don't think it makes any/a big difference. Sometimes I think ```switch``` statements with enums are just easier to read than ```if - else if - else``` cascades (close to Scala's ```match - case``` statements), even for simple cases, but I guess it doesn't really matter here. So, ```if``` it is :smile: